site/src/docs/components/tag/accessibility.mdx

```diff
@@ -14,6 +14,7 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
 ### Pay attention to
 
 - For interactable Tags you must set a descriptive `aria-label` for assistive technologies so the user understands what happens if the Tag is clicked. For example `aria-label="Move to category: <category name>"`.
+- Remember to add a descriptive aria-label for the delete button which does not have a visible label. It can be for example, "Delete item: Americum".
```

Comment:
I think this is more readable if worded like in the bullet above.
~~It can be for example,~~ => For example `deleteButtonAriaLabel="Delete item: Americum"`.

Reply:
- [x] Change wording